### ISSUE TITLE
Weighted UNet with weighted_dice_and_binary_loss

### ISSUE THREAD

**Issue by vi1729** [kohrah/DSBowl2018#1]
Hi Sasha,

I'm working on Weight U-Net implementation. I'm able to get results for Semantic segmentation. But I'm struggling to incorporate WeightMaps and I want to use Weighted_dice_and_binary_loss.

My query is :
1. how to incorporate weightmaps in Model Definition 
`model = Model(input=inputs, output=conv13)`
2. how to pass the weightmaps to the Loss function 
`def weighted_dice_and_binary_loss(X):
    y_pred, y_true, weights = X`
3. What variables I need to pass for X_Train, Y_Train while executing the Model.fit() method.
`model.fit(
    [X_train, W_train], Y_train
    batch_size = 16 ,
    epochs = 100,
    validation_split = 0.1,
    shuffle = True,
    verbose = 1
)`

I have used the above code but running into errors.
Kindly help me.
